applications/GeoMechanicsApplication/custom_elements/triangle_2d_15.hpp

```diff
@@ -0,0 +1,1753 @@
+#pragma once
```

Comment:
All the files related to geometry should be moved to a different directory, for example "custom_geometries"

Reply:
See the comment above. They will be moved to Kratos core